app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        person_instance = Person.people[person["name"]]
+        if person.get("wife"):
+            person_instance.wife = Person.people[person["wife"]]
+        elif person.get("husband"):
+            person_instance.husband = Person.people[person["husband"]]
```

Comment:
As with the previous comments, accessing `Person.people[person['husband']]` assumes that the husband key exists in the dictionary. This could raise a KeyError if the name is not found. Consider adding a check to ensure the key exists before accessing it.